lib/api/tests/js_instance.rs

```diff
@@ -552,7 +552,7 @@ mod js {
         assert_eq!(exported.call(&mut store, &[Val::I32(4)]), Ok(expected));
 
         // It works if we update the memory
-        memory.uint8view(&store).set_index(0, 3);
+        memory.view(&store).uint8view().set_index(0, 3);
```

Comment:
Let's try to use the MemoryView API directly, so we can remove the `uint8view` dependency

Reply:
done